iidm/iidm-tck/src/test/java/com/powsybl/iidm/network/tck/AbstractBusBreakerTest.java

```diff
@@ -261,4 +261,13 @@ public void testNodeBreakerNonSupportedMethods() {
         assertThrows(PowsyblException.class, () -> nodeBreakerView.getNodeInternalConnectedToStream(2), "Not supported in a bus breaker topology");
         assertThrows(PowsyblException.class, () -> nodeBreakerView.getNodesInternalConnectedTo(3), "Not supported in a bus breaker topology");
     }
+
+    @Test
+    void testFictitiousP0AndFictitiousQ0ForInvalidatedBus() {
```

Comment:
Could you move the new tests out of the TCK?
Since the invalidation mechanism is in the IIDM implementation and not in the API, some implementations may update the buses without invalidating them. For instance, in the `powsybl-network-store` implementation, there is no invalidation mechanism and it is possible to call `getP()` and `getQ()` after opening the switch without getting an exception.

Reply:
Note: If we want to enforce the bus invalidation for every implementations, this should be considered as a kind of breaking change and it should be handled in a distinct PR.

Reply:
Sorry, I should have given more details.
You could have just moved your tests in our implementations of the TCK test classes:

- `AbstractBusBreakerTest` -> `BusTest`;
- `AbstractNodeBreakerTest` -> `NodeBrekerTest`.

This way, the tests are sorted by topology kind and you don't have to copy the network creation methods (the scope of the existing methods can be set to protected).

Reply:
Thanks!